packages/block-editor/src/components/block-mobile-floating-toolbar/styles.native.scss

```diff
@@ -12,7 +13,7 @@ $floating-toolbar-height: 44;
 	align-items: center;
 	justify-content: center;
 	align-self: center;
-	margin-bottom: 8px;
+	bottom: 51;
```

Comment:
This is temporary to set proper space of 8px between bottom toolbar and FloatingToolbar. Will refactor soon

Reply:
fixed!